build.sh

```diff
@@ -30,14 +32,19 @@ elif [ "$1" == "release" ]; then
     # remove release from options
     shift
     mkdir release
-    git clone https://github.com/pcengines/coreboot.git -b $2
+    git clone https://review.coreboot.org/coreboot.git release/coreboot
+    cd release/coreboot
+    git submodule update --init --checkout
+    git remote add pcengines https://github.com/pcengines/coreboot.git
+    git fetch pcengines
+    git checkout $1
```

Comment:
$1==release, $2 will be tag|branch according to `./build.sh release <tag|branch> <platform> `. It should be corrected